paimon-format/src/main/java/org/apache/paimon/format/avro/FieldReaderFactory.java

```diff
@@ -120,31 +129,63 @@ public FieldReader visitTimestampMillis(int precision) {
         return TIMESTAMP_MILLS_READER;
     }
 
+    @Override
+    public FieldReader visitTimestampMillis() {
```

Comment:
You can add change origin method `visitTimestampMillis(@Nullable Integer precision)`.